src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -6,19 +6,118 @@
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size = 0;
+    private int currentCapacity;
+    private Node<K, V>[] hashMapList;
+
+    public MyHashMap() {
+        currentCapacity = DEFAULT_CAPACITY;
+        hashMapList = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int index = 0;
+        if (key != null) {
+            index = Math.abs(key.hashCode()) % currentCapacity;
+        }
+        Node<K, V> newElement = new Node<>(key, value);
+        if (hashMapList[index] != null) {
+            Node<K, V> element = hashMapList[index];
+            do {
+                if ((key == null || element.key == null || element.key.hashCode() == key.hashCode())
+                        && ((element.key == key) || (key != null && key.equals(element.key)))) {
+                    element.value = value;
+                    return;
+                }
+                if (element.nextItem == null) {
+                    break;
+                }
+                element = element.nextItem;
+            } while (true);
```

Comment:
if you declare correct condition in while instead of "true" you won't need to use break and it will make code shorter